backend/src/firmware/firmware.constants.ts

```diff
@@ -0,0 +1,33 @@
+// In the format of [owner, [repo, [branches]]]
+export const AVAILABLE_FIRMWARE_REPOS = new Map<string, Map<string, string[]>>([
```

Comment:
There has to be a better way to do this. Already using the Map object tells me something is wrong.

```
AVAILABLE_FIRMWARE_REPOS = {
   'SlimeVR': {
       'SlimeVR-Tracker-ESP': ['main', 'whatever']
   },
   'other user': {}

}
```

i would use object keys instead. it makes for easier reading imo.